apps/air-discount-scheme/web/i18n/withLocale.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { NextComponentType } from 'next'
-import { BaseContext, NextPageContext } from 'next/dist/next-server/lib/utils'
+import { BaseContext, NextPageContext } from 'next/dist/shared/lib/utils'
```

Comment:
Why are these changes needed if we didn't change the nextjs dependency?

Reply:
I don't think this ever worked, its broken on main f.ex, `lib/utils` simply does not exist in `next-server` in this version of next.
I just took the liberty of fixing it here.